packages/local_auth/local_auth_android/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.0.1
+
+* Fixes `getEnrolledBiometrics` to match documented behaviour: 
+  Present biometrics that are not enrolled are no longer returned.
+* `getEnrolledBiometrics` now only returns `weak` and `strong` biometric types.   
+
```

Comment:
You can add that `deviceSupportsBiometrics` now returns the correct value regardless of the enrollment state.